src/base/Algorithms.hh

```diff
@@ -31,5 +31,16 @@ CELER_CONSTEXPR_FUNCTION const T& max(const T& a, const T& b)
     return (b > a) ? b : a;
 }
 
+//---------------------------------------------------------------------------//
+
```

Comment:
Tiny consistency quibble (this is a carryover of how Shift and its stylistic progenitors were formatted) but there shouldn't be a newline here.